docs/ck-tutorial.rst

```diff
@@ -490,6 +490,52 @@ shock tube at Stanford.
     ...
 
 
+Converting from ReSpecTh
+------------------------
+
+PyKED provides converter functions from (and to) the ReSpecTh XML format
+[Varga2015a]_, [Varga2015b]_, though ChemKED files created in this manner may
+require manual edits to fully satisfy our schema.
+
+Given a ReSpecTh XML file ``file.xml``, a corresponding ChemKED file can be created
+either from the command line
+
+.. code-block:: bash
+
+    python converter -i file.xml -o file.yaml
```

Comment:
By defining the entry point, you don't need to invoke Python at all (and I believe what you have in the docs will cause an error because it can't find a script called converter). The entry point is a bash/batch script. You can also invoke a particular module (I think if it has the `__main__` function defined?) by doing `python -m module.name args`, but that's separate from the entry point.

I think `respth2ck` implies XML to YAML, but as I said above, I think its worth it to have 2 explicit scripts and one combined script. Maybe `respth2ck`, `ck2respth`, and `convert_ck`?